src/gluonts/torch/modules/scaler.py

```diff
@@ -38,12 +38,20 @@ class MeanScaler(nn.Module):
 
     @validated()
     def __init__(
-        self, dim: int, keepdim: bool = False, minimum_scale: float = 1e-10
+        self,
+        dim: int,
+        keepdim: bool = False,
+        default_scale: Optional[float] = None,
```

Comment:
Maybe this could be added to the docstring above?

Reply:
Done! I think the docstring of minimum_scale was wrong. Updated it as well. Please take a look.